### PR TITLE
Put format definition display name into a resource

### PR DESCRIPTION
**Pull request by drognanar**

### CODE REVIEW

**Comment by drognanar** [comment]
@tmat

File: src/Microsoft.VisualStudio.InteractiveWindow/Output/OutputClassifierProvider.cs [hunk 1]
```diff
@@ -99,6 +99,7 @@ internal sealed class ErrorOutput : ClassificationFormatDefinition
 
                 public ErrorOutput()
                 {
+                    this.DisplayName = InteractiveWindowResources.ErrorOutputFormatDefinitionDisplayName;
```
Comment (tmat) [inline src/Microsoft.VisualStudio.InteractiveWindow/Output/OutputClassifierProvider.cs]: Is the Name constant still used?

Comment (drognanar) [inline src/Microsoft.VisualStudio.InteractiveWindow/Output/OutputClassifierProvider.cs]: Yes, the are still class level attributes: `[ClassificationType(ClassificationTypeNames = Name)]`, `[Name(Name)]`

**Comment by tmat** [comment]
Superseded by https://github.com/dotnet/interactive-window/pull/42

**Pull request closed without merging**